lib/pleaserun/detector.rb

```diff
@@ -25,14 +25,19 @@ def detect_platform
 
   def detect_systemd
     # Expect a certain directory
-    return false unless File.directory?("/lib/systemd/system")
+    return false unless File.directory?("/lib/systemd/system") || File.directory?("/usr/lib/systemd/system")
```

Comment:
I just checked and Ubuntu 16.04 doesn't seem to have /usr/lib/systemd/system but /lib/systemd/system.
Wasn't systemd supposed to remove this kind of "diversity" between distros? :-/